examples/whisper.android.java/app/src/main/java/com/whispercpp/java/whisper/CDEUtils.java

```diff
@@ -0,0 +1,368 @@
+/*
+
+Copyright (c) Project KanTV(www.cde-os.com). 2019-. All rights reserved.
```

Comment:
Thank you for the interest, but adding 3rd party code to `whisper.cpp` is not desired

Reply:
thanks for your comments.